apps/docs/app/ui/components/propTable/PropTableRender.tsx

```diff
@@ -16,32 +16,53 @@ export interface Item {
     type: ReactNode;
     defaultValue: string;
     description: ReactNode;
+    required: boolean;
 }
 
+const ColoredDefaultValue: React.FC<ColoredDefaultValueProps> = ({ defaultValue }) => {
+    const isBoolean = defaultValue === "true" || defaultValue === "false";
+    const formattedValue = isBoolean ? defaultValue.toString() : `'${defaultValue}'`;
```

Comment:
couldn't you just do .toString and not check if it's a boolean?

Reply:
On top of my head at this point everything is already a "string"